components/availability/src/Availability.svelte

```diff
@@ -317,34 +329,26 @@
               class="slot {slot.selectionStatus} {slot.availability}"
               data-start-time={new Date(slot.start_time).toLocaleString()}
               data-end-time={new Date(slot.end_time).toLocaleString()}
-              on:mouseover={() =>
-                console.log(
-                  "TODO: temp; ",
-                  new Date(slot.start_time).toLocaleString(),
-                )}
+              disabled={slot.availability === "busy"}
               on:click={() => {
-                slot.selectionStatus = handleTimeSlotClick(slot);
+                slot.selectionStatus =
+                  slot.selectionStatus === "selected"
+                    ? "unselected"
+                    : "selected";
+                toggleSelectedTimeSlots(slot);
+                if (!allow_booking) {
+                  dispatchEvent("timeSlotChosen", { timeSlots: slot });
+                }
               }}
             />
           {/each}
         </div>
       </div>
     {/each}
   </div>
-  {#if click_action === "verify"}
-    <footer class="confirmation">
-      Confirm time?
-      <button
-        disabled={!slotSelection.length}
-        on:click={() => {
-          slotSelection.forEach((selectedSlot) => {
-            sendTimeSlot(selectedSlot);
-            selectedSlot.selectionStatus = "unselected";
-            slotSelection = [];
-          });
-        }}
-        class="confirm-btn">Yes</button
-      >
-    </footer>
+  {#if slotSelection.length && allow_booking}
+    <button type="button" on:click={() => sortAndSetEvent(sortedSlots)}
```

Comment:
When `show_tricks` is true, our `main` has two columns; this button occupies, and widens, the first / ticks column when it's present.

We should give this a `grid-column: -1 / 1` to ensure it spans the full width of main. Future styling considerations can make this look however we want, but this'll be a good step toward it.